lib/phobos.rb

```diff
@@ -77,16 +78,20 @@ def configure_logger
         appenders << Logging.appenders.file(log_file, layout: json_layout)
       end
 
-      @ruby_kafka_logger = nil
+      @ruby_kafka_logger = config.custom_kafka_logger
 
-      if ruby_kafka
```

Comment:
If you would like to update the changelog, that would be appreciated

Reply:
Finally, CodeClimate reported some complaints, is this something you would be up to have a look at?

Reply:
sure, I'll do all the housekeeping around this PR if everything else looks good. 👍